### PR TITLE
Build a test_suite of disabled tests

### PR DESCRIPTION
**Pull request by elliottt**
Tests that are disabled are still registered, but are marked as "manual", and added to a test suite of the same name with "_disabled" appended.

### Motivation

Allow disabled pipeline tests to still be run in bazel, but not be run automatically with `bazel test //...`. Additionally there is another test suite added with the "_disabled" suffix when disabled tests are present.

### Test plan

See included automated tests.

### CODE REVIEW

File: test/BUILD [hunk 1]
```diff
@@ -168,16 +170,6 @@ test_corpus_lsp_tests = pipeline_tests(
...
-    tests = test_corpus_tests,
-)
-
-test_suite(
-    name = "test_corpus_lsp",
-    tests = test_corpus_lsp_tests,
-)
-
 test_suite(
     name = "test",
```
Comment (ptarjan) [inline test/BUILD]: Maybe do all of them for concistwncy?

File: test/pipeline_test.bzl [hunk 2]
```diff
@@ -38,6 +55,18 @@ def pipeline_tests(all_paths, test_name_prefix, filter = "*", extra_args = [], t
...
+            tags = tags + extra_tags,
+        )
+
+    if enabled_tests:
+        native.test_suite(
+            name = suite_name,
+            tests = enabled_tests,
+        )
+
+    if disabled_tests:
```
Comment (DarkDimius) [inline test/pipeline_test.bzl]: why not always?

File: test/pipeline_test.bzl [hunk 3]
```diff
@@ -38,6 +55,18 @@ def pipeline_tests(all_paths, test_name_prefix, filter = "*", extra_args = [], t
                 "{}/{}*".format(path, prefix),
             ]) + ["test_corpus_sharded"],
             size = "small",
-            tags = tags,
+            tags = tags + extra_tags,
+        )
+
+    if enabled_tests:
```
Comment (DarkDimius) [inline test/pipeline_test.bzl]: why not always?

File: test/BUILD [hunk 4]
```diff
@@ -168,16 +170,6 @@ test_corpus_lsp_tests = pipeline_tests(
     filter = "LSPTests",
 )
 
-test_suite(
```
Comment (DarkDimius) [inline test/BUILD]: why was this deleted?

File: test/BUILD (hunk 1 above)
Comment (elliottt) [reply test/BUILD]: That last test suite is just the fast `bazel test test` target.

File: test/pipeline_test.bzl (hunk 2 above)
Comment (elliottt) [reply test/pipeline_test.bzl]: That's a fair point. I'll remove the logic to conditionally emit the test suites.
